components/Header/Header.js

```diff
@@ -0,0 +1,46 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+const StyledHeader = styled.header`
```

Comment:
I recommend using an `h1` element here, as it is more semantically appropriate for the `main` heading of a page. This approach ensures that every page on your website has a proper `h1` heading, which is essential for both accessibility and SEO. It helps search engines and screen readers understand the structure and importance of the content on your pages, providing a better experience for all users. 🙂